lxd/instance/drivers/driver_qemu.go

```diff
@@ -6509,6 +6504,11 @@ func (d *qemu) MigrateSend(args instance.MigrateSendArgs) error {
 		return err
 	}
 
+	// Check for stateful support.
+	if args.Live && shared.IsFalseOrEmpty(d.expandedConfig["migration.stateful"]) {
```

Comment:
Please can you expand the commit message to explain what this fixes.

Reply:
Done!